CMakeLists.txt

```diff
@@ -132,6 +132,7 @@ add_subdirectory(omrtrace)
 add_subdirectory(omr)
 add_subdirectory(third_party)
 add_subdirectory(omrsigcompat)
+add_subdirectory(omr_static_lib)
```

Comment:
Should this be happening as one of the last directories to add since it relies on the gc directory, etc?  I apologize for my lack of CMake knowledge in this area.  It is very possible CMake just builds up the dependencies and things will happen in the right order.